docs/src/global/utils.js

```diff
@@ -0,0 +1,9 @@
+export const REPLACE_BACKTICKS_WITH_TAG = (string) => {
```

Comment:
Same question: Should this function name be in UPPSER_SNAKE_CASE?

Also, I was going to ask, don't we need an `// eslint-disable-next-line import/prefer-default-export` up in here?  But it looks like maybe that's turned off in `.eslintrc` so, possibly we don't need it!

Reply:
Fixed.

Reply:
Resolving.